fhirstore/fhirstore.py

```diff
@@ -142,11 +143,21 @@ def create(self, resource: Union[Dict, FHIRAbstractModel]):
                     for err in e.errors()
                 ]
             )
+        except NotSupportedError as e:
+            return self.format_error([str(e)], code="not-supported")
```

Comment:
What do you think about having a `format` method on `NotSupportedError`, `BadRequestError`, etc.?
We could fill stuff like code, etc. inside and return e.format() here (and remove format_error)

Reply:
good idea, I didn't like this format_error anyway

Reply:
done (much better thanks)